tests/documentation/test_files/DocumentationStyleBear/bad_file.py.test.correct

```diff
@@ -1,4 +1,5 @@
 def improper_indents(indents):
+
```

Comment:
@damngamerz , `def` have different padding rules to `class`.  This is distinction is common in styles of other languages also.